src/app.cpp

```diff
@@ -90,6 +90,42 @@ QDir UserScriptDir()
 	return userPath;
 }
 
+std::vector<QDir> CustomUserScriptDirs()
+{
+	std::vector<QDir> vCustomUserScriptDirs;
+
+	if(UserDataDir().exists("custom_user_script_dirs")){
+		QFile inFile((app::UserDataDir().absoluteFilePath("custom_user_script_dirs")));
```

Comment:
Instead of maintaining a custom file and parsing lines from that, it would probably much more convenient to simply use Qt's `QSettings` object and its `value(...)` method with a key "customUserScriptPaths" and a `QStringList` value. Each string in that list corresponding to a user path:

```
QSettings settings;
QStringList userPaths = settings.value ("customUserScriptPaths").toStringList ();
```